lib/ecto/adapters/clickhouse/structure.ex

```diff
@@ -15,8 +15,6 @@ defmodule Ecto.Adapters.ClickHouse.Structure do
     end
   end
 
-  # TODO include views
-
```

Comment:
Is it not possible to get views info? https://clickhouse.com/docs/en/sql-reference/statements/create/view

Reply:
Ah, I saw you commit messages.